tests/TestPsci/TestEnv.hs

```diff
@@ -57,25 +57,38 @@ jsEval = liftIO $ do
     Nothing                      -> putStrLn "Couldn't find node.js" >> exitFailure
 
 -- | Run a PSCi command and evaluate the output with 'eval'.
-runAndEval :: String -> TestPSCi () -> TestPSCi ()
-runAndEval comm eval =
+runAndEval :: String -> TestPSCi () -> (String -> TestPSCi ()) -> TestPSCi ()
```

Comment:
Can you update the documentation here so that it's clear what the new argument does please?